pnc_cli/tools/config_utils.py

```diff
@@ -4,11 +4,11 @@
 import os
 import re
 import string
-from ConfigParser import ConfigParser, NoOptionError, NoSectionError, DuplicateSectionError
+from configparser import ConfigParser, NoOptionError, NoSectionError, DuplicateSectionError
 
-import utils
-from scm_utils import ScmInfo, get_scm_info
-from tasks import Tasks
+# import utils
+# from scm_utils import ScmInfo, get_scm_info
+# from tasks import Tasks
```

Comment:
Don't comment out code, either keep it or remove it.

Reply:
Good catch! We're actually using all those imports. It should be commented out

Reply:
right. mistake. when you will remove the comment, you must change the import to conform to python 2 *and* python 3 rules at the same time (ATM it works only in python 2 but not in 3) - eg.  "import utils" -> "from . import utils"